kpi/renderers.py

```diff
@@ -24,7 +24,7 @@ class AssetJsonRenderer(renderers.JSONRenderer):
 
 class MediaFileRenderer(renderers.BaseRenderer):
     media_type = '*/*'
-    format = None
+    format = 'TODO'
```

Comment:
`drf_spectacular` requires `format` to be a string, otherwise it crashes. What would the right format here?